precise-revoke-redis-sessions/redis.lua

```diff
@@ -0,0 +1,24 @@
+local cursor = 0
+local dels = 0
+local user = ARGV[1]
+
+if string.len(user) < 5 then
+  return "user too short"
+end
```

Comment:
Also is this check still necessary?

Reply:
I think so, it could prevent a typo when calling the script